src/DirectGateway.php

```diff
@@ -43,6 +44,16 @@ public function setSimulatorMode($value)
         return $this->setParameter('simulatorMode', $value);
     }
 
+    public function getReferrerId()
+    {
+        return $this->getParameter('referrerId');
+    }
+
+    public function setReferrerId($value)
+    {
+        return $this->setParameter('referrerId', $value);
+    }
+
```

Comment:
I see the precedence for it, but I don't know that that's a legacy I want to maintain moving forward. I'll mull it over while I sip some coffee and get back to you this afternoon.